class/class.2.1/hello.cpp

```diff
@@ -0,0 +1,83 @@
+#include <iostream>
+#include <fstream>
+#include <unordered_map>
+#include <string>
+
+const std::string FILENAME = "user.txt";
+
+class UserStats {
+public:
+    void load() {
+        std::ifstream infile(FILENAME);
+        if (!infile) {
+            std::cerr << "Failed to open file for reading.\n";
+            return;
+        }
+
+        std::string name;
+        int count;
+        while (infile >> name >> count) {
+            stats[name] = count;
+        }
+        std::cout << "Loaded stats from file.\n";
+    }
+
+    void save() const {
+        std::ofstream outfile(FILENAME);
+        if (!outfile) {
+            std::cerr << "Failed to open file for writing.\n";
+            return;
+        }
+
+        for (const auto& [name, count] : stats) {
+            outfile << name << " " << count << "\n";
+        }
+        std::cout << "Saved stats to file.\n";
+    }
+
+    void reset(const std::string& name) {
+        if (name == "bread") {
```

Comment:
it is better to have a named constexpr  constant in anonymous namespace. the same for FILENAME, delete command